app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -60,6 +81,11 @@ const mapDispatchToProps = dispatch => ({
 SearchAndAddOrder.propTypes = {
   fetchInpatientCareSetting: PropTypes.func.isRequired,
   fetchOutpatientCareSetting: PropTypes.func.isRequired,
+  drug: PropTypes.shape({}),
+};
+
+SearchAndAddOrder.defaultProps = {
+  drug: undefined,
```

Comment:
Not sure setting a prop to 'undefined' is proper practice, how about giving it an empty state e.g: {}